File: KleurigP/KleurGebruik.py
from kleurig import kleur
import kleurig

wit = kleur()
paars=kleur(255,0,255)
oranje=kleur(255,128,0)
lichtgrijs = kleur(180)
donkergrijs=kleur(60)

x = oranje.grijswaarde()
oranjeInZwartwit = kleur(x)

oranje.maakDonkerder()
s = str(oranje)
teruggelezen = kleur(s)

donkerpaars = paars.donkerdereVersie()
donkergeel = kleurig.geel.donkerdereVersie()

print(f"Wit: {wit}");
print(f"Lichtgrijs: {lichtgrijs}");
print(f"DonkerOranje: {oranje}");
print(f"TerugGelezen: {teruggelezen}");
print(f"Paars: {paars}");
print(f"DonkerPaars: {donkerpaars}");
print(f"DonkerGeel: {donkergeel}");

File: KleurigC/KleurGebruik.cs
using System;

Kleur wit = new Kleur();
Kleur paars = new Kleur(Kleur.Maximaal, 0, Kleur.Maximaal);
Kleur oranje = new Kleur(255, 128, 0);
Kleur lichtgrijs = new Kleur(180);
Kleur donkergrijs = new Kleur(60);

byte x = oranje.Grijswaarde();
Kleur oranjeInZwartwit = new Kleur(x);

oranje.MaakDonkerder();
string s = oranje.ToString();
Kleur terugGelezen = new Kleur(s);

Kleur donkerPaars = paars.DonkerdereVersie();
Kleur donkerGeel = Kleur.Geel.DonkerdereVersie();

Console.WriteLine($"Wit: {wit}");
Console.WriteLine($"Lichtgrijs: {lichtgrijs}");
Console.WriteLine($"DonkerOranje: {oranje}");
Console.WriteLine($"TerugGelezen: {terugGelezen}");
Console.WriteLine($"Paars: {paars}");
Console.WriteLine($"DonkerPaars: {donkerPaars}");
Console.WriteLine($"DonkerGeel: {donkerGeel}");